src/plugins/services.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import Vue from 'vue'
+import ServicesPlugin from '@/services/plugin'
+
+// Load the plug-in
+Vue.use(ServicesPlugin, {
+  offline: process.env.VUE_APP_SERVICES === 'offline'
+})
```

Comment:
This is similar to the plug-in from #504, and to the other plug-ins in the project (following what was defined in the Tim Creative's theme). The plug-in is defined in `src/services/plugin.js`, and this file just loads it.

The `plugins/index.js` imports each installable plug-in file (such as this one).